### ISSUE TITLE
added the addon and uhhhhhhh

### ISSUE THREAD

**Issue by Chancellor2020** [dimdenGD/OldTwitter#975]
![image](https://github.com/user-attachments/assets/73e7ede4-3af9-4473-839b-c5acdc247262)

There was an error in initializing security header generator: Error: Failed to fetch homepage: TypeError: https:/// is not a valid URL.
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/115.0
OldTwitter doesn't allow unsigned requests anymore for your account security. Currently the main reason for this happening is social network tracker protection blocking the script. Try disabling such settings in your browser and extensions that do that and refresh the page. This can also happen if you're not logged in.

I'm currently on the developer version of 115.0 Firefox (The version thats compatible for an aero mod)

It pretty much happens everywhere on Twitter, is there a way to fix this?

**1.8.9.6**
Can be seen on bottom-right

**Comment by Chancellor2020** [comment]
![image](https://github.com/user-attachments/assets/04d5b7bc-8e54-4e49-bfd7-0ae598bd160e)
And after I press OK, it just stays locked at this screen

**Comment by 2lipp2p** [comment]
happening to all firefox users, still waiting on fix

**Comment by dimdenGD** [comment]
Works fine with Firefox 133.0

**Comment by Chancellor2020** [comment]
couldn't switch to that, it would cause the echelon skin on my firefox to break

**Comment by 2lipp2p** [comment]
vouch!! i have the same skin

**Comment by MegaTeam89** [comment]
Please fix this for ESR 115

**Comment by FanACYT** [comment]
> couldn't switch to that, it would cause the echelon skin on my firefox to break. To get one thing, I would need to sacrifice another, and tbh I'd rather not fw that 😭 💀

yeah i cant really get out of 115 esr bec im using it on a win 7 machine and im not gonna install things for extended kernels for newer apps on it

**Comment by dimdenGD** [comment]
Try latest version, maybe fixed

**Comment by bobbuscus** [comment]
> Try latest version, maybe fixed

not fixed, just forever is stuck on the loading screen now, but the error is gone

(dont know if im doing this right srry)
![ex](https://github.com/user-attachments/assets/227e7868-fb48-4894-95f0-b99c12b8c54b)

**Comment by dimdenGD** [comment]
ngl no idea at this point then

**Comment by 2lipp2p** [comment]
![image](https://github.com/user-attachments/assets/356b2c48-9633-426b-a310-2ecf61a6a4a1)

**Comment by 2lipp2p** [comment]
heres console if it helps 
![image](https://github.com/user-attachments/assets/ac2ad52a-f174-4e70-83bd-906d253644ca)

**Comment by TheAGames10** [comment]
> Works fine with Firefox 133.0

Just wanted to mention that this is also happening with Chromium, as the current version of Opera GX i'm using is getting the same error, I have no idea how to fix it or if there is even a fix for it.
